client/keys/delete.go

```diff
@@ -53,7 +53,7 @@ private keys stored in a ledger device cannot be deleted with the CLI.
 					return err
 				}
 
-				if info.GetType() == keyring.TypeLedger || info.GetType() == keyring.TypeOffline {
+				if k.GetType() == keyring.TypeLedger || k.GetType() == keyring.TypeOffline {
```

Comment:
I would like to remove `GetType` func from `Record` entirely. I think, we don not need that, do we?